src/Morphic-Base/MorphicWindowManager.class.st

```diff
@@ -185,8 +185,9 @@ MorphicWindowManager >> openNewWindowWithRoot: aMorph [
 { #category : #saving }
 MorphicWindowManager >> saveAllAttributes [
 	
-	allAttributes := windows collect: [  :w | w attributsSave ].
-]
+
+	allAttributes := windows collect: [  :w | w attributesSave ].
+
```

Comment:
There is a missing closing bracket here. Did you touch it directly in the repo? don't do that!

Reply:
And format this method please :)